metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -114,7 +114,6 @@
 import static org.apache.kafka.common.protocol.Errors.NO_REASSIGNMENT_IN_PROGRESS;
 import static org.apache.kafka.common.protocol.Errors.UNKNOWN_TOPIC_ID;
 import static org.apache.kafka.common.protocol.Errors.UNKNOWN_TOPIC_OR_PARTITION;
-import static org.apache.kafka.controller.ConfigurationControlManager.NO_OP_EXISTENCE_CHECKER;
```

Comment:
@cmccabe I think we can remove this static field from `ConfigurationControlManager`. I don't think we use it or need it anymore.